project/src/main/webapp/user-home.jsp

```diff
@@ -19,5 +29,11 @@
         <docs-component></docs-component>
       </div>
     </div>
+
+    <script>
+      function loadDocument() {
+        window.location.href = "/Document?documentHash=" /* documentHash */;
```

Comment:
modify this to use the documentHash url attribute; and probably add a null check where if the hash doesn't exist, redirect back to user home